controller_manager:
  ros__parameters:
    update_rate: 1000 # Hz
    
    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
      
    effort_controller:
      type: effort_controllers/JointGroupEffortController

joint_state_broadcaster:
  ros__parameters:
    joints:
      - hip_to_thigh
      - thigh_to_shin
      - slider_joint

# Effort controller configuration
effort_controller:
  ros__parameters:
    joints:
      - hip_to_thigh
      - thigh_to_shin
      - slider_joint
    command_interfaces:
      - effort
    state_interfaces:
      - position
      - velocity
      - effort
